package.json

```diff
@@ -44,11 +45,17 @@
   },
   "devDependencies": {
     "@babel/node": "^7.2.2",
+    "babel-jest": "^24.5.0",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.11.2",
     "eslint": "^5.15.3",
     "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-react": "^7.12.4",
+    "jest": "^24.5.0",
     "mongodb-memory-server": "^4.2.1",
     "nodemon": "^1.18.10",
-    "prettier": "^1.16.4"
+    "prettier": "^1.16.4",
+    "react-addons-test-utils": "^15.6.2",
```

Comment:
It seems we don't need it. I uninstalled it and testing still seems to work with no errors.

Reply:
@hitochan777 Can we call this PR "Added Jest and Enzyme for React Component testing"? and merge it so others can use this functionality. Then we can turn your request for a change to a new request for an enhancement.? Or is it better to keep things this way for now?
Sorry if this is a weird question. I'm still getting used to this.

Reply:
We can treat this PR as the one for react component testing.
In my opinion (though I've never worked as a professional software engineer), it is always better to keep a PR small so that it does not overwhelm reviewers.